npps4/webui/endpoints/unlock_backgrounds.py

```diff
@@ -0,0 +1,60 @@
+import fastapi
+
+from .. import template
+from ...app import webui
+
+import sqlalchemy
+
+import npps4.db.main
+import npps4.db.item
+import npps4.idol
+import npps4.scriptutils.user
+from npps4.db.item import Background
+
+import npps4.idol
+import npps4.idol.system.background
+import npps4.db.main
+import npps4.db.item
+import npps4.scriptutils.user
```

Comment:
Relative import for `npps4` package.

Reply:
Oh one more thing, specific to `from npps4.db.item import Background` line. Import the whole module, not single function from it.

```
from ...db import item
```